Update TravelTime API spec to version 1.3.0

Donatas-L opened the pull request:
Updated the TravelTime API specification to version 1.3.0. Added detailed descriptions for various endpoints and components, including new tags for better organization.

Donatas-L committed:
Update TravelTime API spec to version 1.3.0

Updated the TravelTime API specification to version 1.3.0. Added detailed descriptions for various endpoints and components, including new tags for better organization.

Donatas-L committed:
Add descriptions to request properties in YAML

Added descriptions for various request properties including route searches, time filter searches, and snapping behavior.

arnasbr committed:
developing

arnasbr committed:
developing

arnasbr commented on traveltime-spec.yml:
```diff
@@ -1606,338 +2306,718 @@ components:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/RequestH3Fast'
+      responses:
+        '200':
+          description: Successful response
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ResponseH3Response'
+        '400':
+          description: Bad request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+        '401':
+          description: Unauthorized
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
```
I added some of the main error codes that are specific to our API. Before we didn't have them, so for simplicity we can get rid of them from this PR

arnasbr committed:
fix postcodes

arnasbr committed:
Merge pull request #8 from traveltime-dev/fix-postcodes

arnasbr committed:
Add GET requests (time-map, time-filter and routes)

arnasbr committed:
fix time-filter GET response

arnasbr committed:
fix routes

arnasbr committed:
Merge pull request #9 from traveltime-dev/add-get

Add GET requests (time-map, time-filter and routes)

fildawtraveltime committed:
fix: time-filter/fast POST response, default values for RequestTransportation, various descriptions

AivarasPuskorius committed:
Updated request transportation for distance-map

AivarasPuskorius committed:
Updated H3 and Geohash coords

- Geohash and H3 accept either `lat` and `lng` or `geohash_centroid`/`h3_centroid` respectively

AivarasPuskorius committed:
Adjusting description

AivarasPuskorius committed:
Merge pull request #12 from traveltime-dev/h3-geohash-coords-update

Updated H3 and Geohash coords

fildawtraveltime committed:
chore: description

fildawtraveltime committed:
fix: h3 snapping attributes

fildawtraveltime committed:
fix: snapping for geohash

AivarasPuskorius committed:
Merge pull request #11 from traveltime-dev/distance-map-transportation

Updated request transportation for distance-map

fildawtraveltime committed:
Merge pull request #10 from traveltime-dev/time-filter-spec-fixes

Fix time-filter/fast POST response

fildawtraveltime committed:
Merge pull request #13 from traveltime-dev/h3-snapping

fix h3 and geohash: remove threshold attribute for snapping
